### PR TITLE
Adding BadRequestHttpException to updateoperationkey

### LINKED ISSUE #725: /v1/update-operation-key service is getting stuck if non-existing operation server uuid is passed as parameter in request body.
**Problem Description :** 

/v1/update-operation-key service is getting stuck if non-existing operation server uuid is passed as parameter in request body.

### PR DESCRIPTION
**Pull request by ManasaBM1**
Fixes #725

### CODE REVIEW

File: server/basicServices/basicServices/BasicServicesService.js [hunk 1]
```diff
@@ -1398,7 +1398,7 @@ exports.updateOperationKey = function (body, user, originator, xCorrelator, trac
             isUpdated = await operationServerInterface.setOperationKeyAsync(operationUuid, newOperationKey);
           }
         } else {
-          throw new Error("OperationServerUuidisnotPresent")
+          reject(new BadRequestHttpException("OperationServerUuidisnotPresent"))
```
Comment (PrathibaJee) [inline server/basicServices/basicServices/BasicServicesService.js:1401]: Shall we make the reson for rejection 
from 
OperationServerUuidisnotPresent
to 
OperationServerUuid is not present

File: server/basicServices/basicServices/BasicServicesService.js [hunk 2]
```diff
@@ -1407,7 +1407,7 @@ exports.updateOperationKey = function (body, user, originator, xCorrelator, trac
             isUpdated = await operationClientInterface.setOperationKeyAsync(operationUuid, newOperationKey);
           }
         } else {
-          throw new Error("OperationClientUuidisnotPresent")
+          reject(new BadRequestHttpException("OperationClientUuidisnotPresent"))
```
Comment (PrathibaJee) [inline server/basicServices/basicServices/BasicServicesService.js:1410]: Shall we make the reson for rejection 
from 
OperationClientUuidisnotPresent
to 
OperationClientUuid is not present

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Thanks for the changes. Some minor changes requested . Please resolve them. Thanks.

File: server/basicServices/basicServices/BasicServicesService.js (hunk 1 above)
Comment (ManasaBM1) [reply server/basicServices/basicServices/BasicServicesService.js]: done

File: server/basicServices/basicServices/BasicServicesService.js (hunk 2 above)
Comment (ManasaBM1) [reply server/basicServices/basicServices/BasicServicesService.js]: done

**Review by PrathibaJee — APPROVED** [review]

**Pull request merged**